packages/presentation/lib/src/base_bloc/base_tile.dart

```diff
@@ -1,23 +1,28 @@
 class BaseTile<D> {
   final bool isLoading;
+  final String? errorMessage;
   final D? tile;
 
   BaseTile({
     required this.tile,
     required this.isLoading,
+    required this.errorMessage,
   });
 
   factory BaseTile.init() => BaseTile<D>(
-        isLoading: false,
+        isLoading: true,
```

Comment:
why did u change it?

Reply:
hmm, I decided that, when is page created, it waits data anyway...

Reply:
not always. for ex, if we won't fetch actors from details page, we shouldn't load any data. it is already passed via arguments